modules/s2i/core/artifacts/opt/jboss/container/s2i/core/s2i-core

```diff
@@ -33,6 +33,7 @@ function s2i_core_env_init() {
   S2I_IMAGE_SOURCE_MOUNTS="${S2I_IMAGE_SOURCE_MOUNTS:-${CUSTOM_INSTALL_DIRECTORIES}}"
   S2I_ENABLE_INCREMENTAL_BUILDS="${S2I_ENABLE_INCREMENTAL_BUILDS:-true}"
   S2I_DELETE_SOURCE="${S2I_DELETE_SOURCE:-true}"
+  S2I_ENABLE_JLINK="${S2I_ENABLE_JLINK:-true}"
```

Comment:
I think it's best to default this to false (as documented in `modules/s2i/core/module.yaml`)